server/controllers/mentionController.js

```diff
@@ -1,17 +1,78 @@
-// const asyncHandler = require("express-async-handler");
 const Mention = require("../models/mentionModel");
 
 // @desc    Get Mentions
-// @route   GET /api/mentions/
+// @route   GET /api/mentions?platforms=xxx&keyword=xxx&sort=xxx
```

Comment:
Take a look at JSDoc @param, you should be able to define the individual query parameters instead of including them in the route like this

Reply:
@jessdesr  First time ever I use this, so not too sure if I got it right